tutorial/Lab3/README.md

```diff
@@ -0,0 +1,82 @@
+# Lab 3. Keeping track of the deployed application
+
+Lets say you have deployed different release versions of your application (i.e. you upgraded the running application). How do you keep track of the versions and how can you do a rollback?
+
+# Revision management using Kubernetes directly
+
+In this part of the lab we should illustrate revision management of `guestbook`, using Kubernetes directly. Unfortunately, Kubernetes does not provide any support for revision management. The onus is on you to provide management of your systems and any updates or changes you make.
```

Comment:
In this part of the lab, we'll illustrate revision management of `guestbook`, by using Kubernetes directly. Unfortunately, Kubernetes does not provide any support for revision management. The onus is on you to manage your systems and any updates or changes you make.

Reply:
Updated